docs/src/howto/networking/default-dns.md

```diff
@@ -0,0 +1,86 @@
+# How to use default DNS
+
+Canonical Kubernetes includes a default DNS (Domain Name System) essential for internal communication within your cluster. 
+When enabled, the DNS facilitates service discovery
+by assigning each service a DNS name. 
+
+## What you'll need
+
+This guide assumes the following:
+
+- You are installing on Ubuntu 22.04 or later, **or** another OS which supports
+  snap packages (see [snapd support])
+- You have root or sudo access to the machine
+- You have an internet connection
+- The target machine has sufficient memory and disk space. To accommodate
+  workloads, we recommend a system with ***at least*** 20G of disk space and 4G of
+  memory.
+- You have Canonical Kubernetes installed and a bootstraped cluster. (See: [getting-started-guide](#TODO))
+
+## Is DNS enabled?
+
+Find out wether you have enabled DNS with the following command:
+
+```bash
+sudo k8s status
+```
+
+The default state for the cluster is `dns enabled`.
+
+## Enabling and disabling DNS
```

Comment:
I think we should agree on one format for the those how-to documentations.
For the storage, I used:
1) Enabling
2) Configuration
3) Disabling

I don't have a strong opinion on which one is more preferable but we should be consistent. @evilnick could you say which one you like better and then we update my or this PR accordingly

Reply:
Yeah, I already mentioned this in the comment ^

The Canonical style is to use imperatives rather than progressives, so 'Enable xxx' rather than 'Enabling xxx'
If it helps, imaging sticking 'How to' in front of each heading

Reply:
Thanks! My question was more towards the structure, though.
e.g. should we explain how to enable/disable the component in one section and then how to configure it in another?
Or should we have one section on how to enable the component, then we configure it in the next section and disable it in the last one.
WDYT?

Reply:
I like the suggested structure from Benjamin. Then I do not need to say in the middle, hey re-enable what you just turned off if you would like to continue.

Reply:
@evilnick What ideal structure would you prefer?

Reply:
The new warning makes it clear not to disable dns now, unless you really want to see some flames. SO we could keep this structure also. I'm happy either way :)

Reply:
yes, definitely. I was a bit confused because at one point I'm sure someone mentioned we would have to set the config for things ***before*** enabling them